src/main/java/com/sap/oss/phosphor/fosstars/data/github/GoSecDataProvider.java

```diff
@@ -33,20 +28,38 @@
  * </ul>
  */
 public class GoSecDataProvider extends AbstractStaticScanToolsDataProvider {
-
-  /** 
-    * A step in a GitHub action that triggers analysis with GoSec.
-    */
-  private static final Pattern RUN_STEP_GOSEC_REGEX_PATTERN =
-      Pattern.compile("^.*securego/gosec.*$", Pattern.DOTALL);
-
-  /** A step config of a GitHub action that triggers GoSec scans with include rule. */
-  private static final Pattern RUN_STEP_GOSEC_WITH_INCLUDE_REGEX_PATTERN =
```

Comment:
Same as above isn't it nice to create these patterns once and use it in the class?